node/node.go

```diff
@@ -873,6 +873,17 @@ func NewNode(config *cfg.Config,
 		return nil, fmt.Errorf("could not create addrbook: %w", err)
 	}
 
+	for _, addr := range splitAndTrimEmpty(config.P2P.BootstrapPeers, ",", " ") {
+		netAddrs, err := p2p.NewNetAddressString(addr)
+		if err != nil {
+			return nil, fmt.Errorf("invalid bootstrap peer address: %w", err)
+		}
+		err = addrBook.AddAddress(netAddrs, netAddrs)
```

Comment:
I've just realize that the source address (second parameter) should not be that one.

In fact, when we manually add addresses to the address book, the source should be the node itself. Refer to the implementation [spec](https://github.com/cometbft/cometbft/blob/main/spec/p2p/implementation/addressbook.md#adding-addresses) (point 3. referring to the source address) and to the way we add addresses of configured persistent peers on the [switch](https://github.com/cometbft/cometbft/blob/main/p2p/switch.go#L503).